tests/contrib/molten/test_molten.py

```diff
@@ -85,18 +87,15 @@ def test_resources(self):
 
     def test_distributed_tracing(self):
         """ Tests whether span IDs are propogated when distributed tracing is on """
-        def prepare_environ(environ):
-            environ.update({
-                'DD_MOLTEN_DISTRIBUTED_TRACING': 'True',
-                'HTTP_X_DATADOG_TRACE_ID': '100',
-                'HTTP_X_DATADOG_PARENT_ID': '42',
+        with override_config('molten', dict(distributed_tracing=True)):
```

Comment:
:+1: might be worth having a test case to ensure that with `distributed_tracing = False` that we do not set the `trace_id` and `parent_id`.